### DISCUSSION TITLE
Mudanças na tributação Simples Nacional em 2018

### DISCUSSION THREAD

**Discussion by fforbeck** [DyegoCosta/trabalhando-remoto#175] [General]
Fala pessoal,

acredito que a maioria já está ciente disso porém gostaria compartilhar e receber algumas ideias. 

Em 2018 teremos mudanças na tributação para empresas do Simples Nacional. Para resumir:
- Até final de 2017 poderemos emitir nota como Suporte Tecnico em Informatica (depende do sistema de NFSe de cada cidade), tal atividade utiliza o anexo III para cálculo do imposto e dependendo do tipo do serviço, tem a possibilidade de isenção de ISS, PIS/COFINS, caindo para 4% de alíquota.
- Em 2018 a atividade de Suporte Tecnico utilizará o Anexo V, o mesmo anexo utilizado pela atividade de Desenvolvimento de programas de computador, assim mudou a tributação. Basicamente a taxa de referencia é de 18%, pode variar dependendo de cada caso. Quem tem empregados e a folha representa + que 28% do faturamento, terá uma boa reducação na alíquota. Porém nós autonomos estamos lascados. Também não está claro se a isenção do ISS, PIS/COFINS poderão ser utilizadas.

Alguém que está como ME no Simples já tem alguma estratégia para reduzir legalmente a alíquota?

Valeu!

**Comment by pablo-sg-pacheco** [comment]
Como não tenho funcionários, a estratégia q vou tentar usar é pagar prolabore pra mim e pros socios no valor de 28% do faturamento e reduzir a alíquota

**Comment by fforbeck** [comment]
Interessante @pablo-sg-pacheco. Eu dei uma olhada nisso, porém pelo que eu entendi o cálculo leva em consideração tua folha de pagamento dos últimos 12 meses, dessa forma teria que fazer isso durante 1 ano para atingir os 28% e então mudar de anexo e reduzir a alíquota. É isso mesmo?

**Comment by pablo-sg-pacheco** [comment]
Eu nem tinha me atentado a isso. Mas se for necessário, então é isso mesmo :)

Além disso vou tentar me pagar somente o suficiente de prolabore para não ter que pagar imposto como pessoa física, ou seja, acho que até 1800 reais seria possível. O resto eu poderia receber como dividendos

**Comment by glaucomonteiro** [comment]
Onde vc viu essa informação?

Pelo que eu lembro da lei, a parte de informatica vai ter uma grande redução de impostos a partir do ano que vem, e é basicamente o que o contabilizei diz tambem, então se puder postar a referencia disso seria interessante pra ver e confirmar o que realmente vai acontecer ano que vem.

https://www.contabilizei.com.br/contabilidade-online/novo-simples-nacional/

**Comment by fforbeck** [comment]
@glaucomonteiro 

> § 5o-> I.  Sem prejuízo do disposto no § 1o do art. 17 desta Lei Complementar, as seguintes atividades de prestação de serviços serão tributadas na forma do Anexo V desta Lei Complementar:   (Redação dada pela Lei Complementar nº 155, de 2016)   

> VI - engenharia, medição, cartografia, topografia, geologia, geodésia, testes, suporte e análises técnicas e tecnológicas, pesquisa, design, desenho e agronomia; 

> XII - outras atividades do setor de serviços que tenham por finalidade a prestação de serviços decorrentes do exercício de atividade intelectual, de natureza técnica, científica, desportiva, artística ou cultural, que constitua profissão regulamentada ou não, desde que não sujeitas à tributação na forma dos Anexos III ou IV desta Lei Complementar.   (Redação dada pela Lei Complementar nº 155, de 2016)    

> § 5o-J.  As atividades de prestação de serviços a que se refere o § 5o-I serão tributadas na forma do Anexo III desta Lei Complementar caso a razão entre a folha de salários e a receita bruta da pessoa jurídica seja igual ou superior a 28% (vinte e oito por cento).    (Incluído pela Lei Complementar nº 155, de 2016)

> § 5o-K.  Para o cálculo da razão a que se referem os §§ 5o-J e 5o-M, serão considerados, respectivamente, os montantes pagos e auferidos nos doze meses anteriores ao período de apuração para fins de enquadramento no regime tributário do Simples Nacional.    (Incluído pela Lei Complementar nº 155, de 2016)   

> § 5o-M.  Quando a relação entre a folha de salários e a receita bruta da microempresa ou da empresa de pequeno porte for inferior a 28% (vinte e oito por cento), serão tributadas na forma do Anexo V desta Lei Complementar as atividades previstas:   (Incluído pela Lei Complementar nº 155, de 2016)      Produção de efeito

> http://www.planalto.gov.br/ccivil_03/leis/lcp/Lcp123.htm#art18§5i

**Comment by glaucomonteiro** [comment]
Essa parte do suporte técnico eu não tinha perstado muita atenção da ultima vez que tinha visto, prestei mais atenção nessa parte, que vai ter a redução do imposto

> § 5o-D.  Sem prejuízo do disposto no § 1o do art. 17 desta Lei Complementar, as seguintes atividades de prestação de serviços serão tributadas na forma do Anexo III desta Lei Complementar: 

> IV - elaboração de programas de computadores, inclusive jogos eletrônicos, desde que desenvolvidos em estabelecimento do optante;

> V - licenciamento ou cessão de direito de uso de programas de computação;

> VI - planejamento, confecção, manutenção e atualização de páginas eletrônicas, desde que realizados em estabelecimento do optante;

**Comment by guilhermepiccoli** [comment]
Pessoal, estou nessa também! Minhas impressões são as seguintes, por favor me corrijam se eu estiver falando besteira:

1 - O Simples 2018 "quebrou as pernas" de quem usava o "truque" do Anexo III. Isso porque, com base nas leis complementares 123 (que descreve o Simples, originalmente) e na 155 (a "nova", que atualizou o Simples em 2018), agora temos a seguintes situação:

Ou a atividade cai no Anexo III, mas se não houver folha de pgto de ao menos 28%, vira Anexo V;
Ou a atividade cai no Anexo V, e como bonus, se houver folha de pagamento de 28%, cai no anexo III.
As atividades a que me refiro são Suporte (CNAE 6209-1) e Desenvolvimento de Programas sob Encomenda (CNAE 6205-1). Ou seja, temos que ter pro-labore de >=28% pra cair no III :(

Eu fiz umas contas, dependendo do salário que a pessoa recebe, e como o pro-labore está sujeito a IRPF e INSS, fica feia a coisa...pode até compensar ficar mesmo no anexo V, as vezes o IRPF do pro-labore + INSS + o imposto do Anexo III superam o imposto sozinho do Anexo V (com pro-labore baixo). Fiquem espertos!


2 - Uma idéia que estou eh fugir do Simples! No Lucro Presumido, supondo que a pessoa tenha isenção de PIS, Cofins e ISS (veja leis Art. 46, III, da IN SRF nº 247/2002 e LEI COMPLEMENTAR Nº 116, DE 31 DE JULHO DE 2003), a taxa eh:
15% de IRPJ
9% de CSLL,
ambas em cima de 32% (o tal Lucro Presumido).

Então a taxa real eh de (0.24*32%) == 7.68% !!! Eh uma taxa menor que o Simples dependendo do caso.
Um "catch" nessa situação:  no Lucro Presumido, INSS em cima do pro-labore eh de (20% + 11%), então um pro-labore de 1000 reais já tem 310 de INSS!

Se alguem notar um erro nos meus cálculos ou assunções, por favor me avise =)
Abs!

**Comment by fforbeck** [comment]
@guilhermepiccoli exatamente. Quebrou as pernas haha.
LP + isenções + pro-labore mínimo acredito que seja a melhor opção, também cheguei nessa alíquota.

**Comment by copercini** [comment]
Acredito que seu cálculo está correto, mas depende da sua renda para ter vantagem no Lucro Presumido. Existe uma [tabelona do contabilizei](http://conteudo.contabilizei.com.br/planilha-simples-nacional-lucro-presumido-2017) que compara simples vs lucro presumido, mas utilizando as regras de 2017.

Fiz algumas simulações utilizando as regras de 2018 (pro-labore de 28% para o simples)

**(Os valores são apenas hipotéticos senhor fiscal da receita federal que olha as redes sociais)**

Para 10k reais mensais no simples 2018:
-Mínimo de R$2.800,00 de pro labore
Nas minhas contas daria **7,46%** (conforme imagem abaixo)
![screenshot_2](https://user-images.githubusercontent.com/21266776/42660460-a61eaf7e-8601-11e8-9371-757504a6fc07.png)


Para 10k reais mensais no lucro presumido:
-Mínimo de R$937,00 de pro labore
-INSS de 11% + 20% de Cota Patronal
-Considerando isenções de PIS, Cofins e ISS
Nas minhas contas daria **12,37%** (conforme imagem abaixo)
![screenshot_2](https://user-images.githubusercontent.com/21266776/34641966-00f1e4d6-f2f3-11e7-95c9-a9445c56f0bd.png)

---------------------
Para 20k reais mensais no simples 2018:
-Mínimo de R$5.600,00 de pro labore
Nas minhas contas daria **10,32%** (conforme imagem abaixo)
![screenshot_3](https://user-images.githubusercontent.com/21266776/42660519-d07c0f00-8601-11e8-8d38-42a35868b5f5.png)

Para 20k reais mensais no lucro presumido:
-Mínimo de R$937,00 de pro labore
-INSS de 11% + 20% de Cota Patronal
-Considerando isenções de PIS, Cofins e ISS
Nas minhas contas daria APENAS 10,03% (conforme imagem abaixo)
![screenshot_4](https://user-images.githubusercontent.com/21266776/34642072-916040e8-f2f4-11e7-9956-d7a30b9ccc07.png)


Conclusão:
Quanto maior a renda mais o lucro presumido vale a pena!

**Comment by guilhermepiccoli** [comment]
Copercini, exatamente! Quanto maior a renda, mais compensa o Lucro Presumido!

No caso de 10k de renda, seria: 7.68% de 10k + (31% de 937) + contador == 768 + 290.47 + contador == 1058.47 + contador
Ou seja, taxa efetiva de 10.59% ao mes + contador.

No caso do simples, pode-se descontar também o ISS e PIS/Cofins! Muita gente não faz isso, mas segundo a resolução CGSN nº 94/2011 (artigo 25-A), podemos sim descontar esses valores no cálculo da aliquota!

Aí, nesse caso, deve-se observar as tabelas do Anexo III na Lei complementar 155, e as porcentagens de cada imposto (como CSLL, Cofins, ISS, etc) na geração da aliquota final do Simples (que no fundo, nao devia se chamar Simples e sim Complexo-Pra-Ca****o heheh).

Então vejamos, 2800 de pro-labore contém (7.5% - 142.8) de IRPF e 11% de INSS -> 67.2 + 308 == 375.2

Agora o Simples: Anexo III, faixa 1 (até 180k por ano), temos aliquota nominal de: 6,00%
Porcentagem disso que se refere aos impostos mandatórios IRPJ, CSLL e CPP: 50.9%.

Como a aliquota eh progressiva agora (pra complicar ainda mais), vamos considerar o primeiro mes, seguindo a formula:
(RBT12 * Aliq - PD) / RBT12, com RBT12 == 10k e PD == 0
Nesse caso, fica 0.509 * (10000*6)/10000 == 3.06%

Assim, imposto total eh: (0.0306*10000) + 375.2 == 681.2 + contador
Isto é, uma aliquota muito boa de 6.82%, que compensa muito(!) em relação ao LP.

Cada um deve avaliar seu caso, quando a pessoa passa a ganhar na Faixa 2 (ou mais) do Simples, cujas aliquotas sao maiores, e o IRPF do pro-labore começa a esfaquear e girar a faca (27.5% !!), o LP compensa.

P.S. Por favor, se fiz algum cálculo errado e alguém notar, me avisem heheh
Valeu!

**Comment by emada** [comment]
Olá pessoal!

Muito interessante essa discussão. Parabéns Dyego pela iniciativa e a todos pela contribuição prestada para a discussão.

Copercini, achei bem interessante a sua adaptação da planilha do Contabilizei. Você poderia compartilhar com a gente? Existe uma forma de fazer um upload por aqui? Caso contrário, você poderia por favor enviar para o meu email? (emerson@fantasticatecnologia.com.br)

Tudo de bom,
Emerson

**Comment by copercini** [comment]
@emada por sorte estava aqui na pasta downloads, essa é a versão que tinha no site, mas é preciso fazer algumas considerações como o pró-labore de 28% e a tabela também parece ter um bug que não calcula o IR para o simples, então é preciso um certo cuidado
[1514893087Simples_Nacional_vs_Lucro_Presumido.xlsx](https://github.com/DyegoCosta/trabalhando-remoto/files/1809056/1514893087Simples_Nacional_vs_Lucro_Presumido.xlsx)

**Comment by clmadrazo** [comment]
@copercini essa arquivo que compartilhou está com as informações atualizadas?

**Comment by copercini** [comment]
@clmadrazo é de 2017, ainda não tem a regra dos 28% de simples 2018

Pelo que conversei com o contador, o simples tbm tem isenção de ISS e PIS/COFINS, então dos 6% viram aproximadamente 3%, mas eu vou confirmar isso esse mês

**Comment by clmadrazo** [comment]
@copercini obrigado

**Comment by copercini** [comment]
Confirmado: o imposto do simples para exportação de serviços são 3,05% (6% com isenção de Cofins, PIS e ISS) na primeira faixa e quando cai no anexo 3, o problema é conseguir cair no anexo 3...

Para conseguir entrar no anexo 3 é preciso que a folha de pagamento ou pro labore seja maior ou igual 28% do faturamento (chamado fator R)

PS. o primeiro mês da empresa sempre será anexo 5 (que paga 10,67% de imposto no caso de exportação de serviços), pois o fator R tem que ser 28% baseados nos últimos 12 meses, e como não teve nem pro labore e nem receita nos meses anteriores a receita considera como anexo 5.

**Comment by copercini** [comment]
Senhores, atualizei o meu cálculo acima utilizando o valor correto do simples (3,05% para 10k e 4,18% para 20k), o lucro presumido começa valer a pena pra quem ganha muita grana, acima de 18k ou 19k!

**Comment by renatonascalves** [comment]
Se alguém pudesse resumir o que já foi debatido até aqui, seria mega útil.

**Comment by copercini** [comment]
Mais uma informação importante, quem for de Lucro presumido precisa fazer cadastro na Siscoserv e fazer as devidas declarações

**Comment by brunobarretosa** [comment]
@copercini Será que, mesmo ganhando mais de 18k-19k não valeria a pena o cara fazer Simples Nacional? A diferença é grande? Pelo pouco que pesquisei, dá mais trabalho ter o Lucro Presumido - que fazer declarações lá pro Siscoserv etc.

Alguém já fez a tabela atualizada comparando Simples vs Lucro Presumido com isenção de PIS, COFINS e ISS e utilizando o Fator R?

To perdidão nesta etapa de decidir qual modelo mais "tranquilo e rentável" da empresa.

Valeu!

**Comment by copercini** [comment]
@brunobarretosa refiz as contas certinho (e conferi com o que o contador me cobra), o simples vale a pena até R$21.000,00 por mês
![image](https://user-images.githubusercontent.com/21266776/54075916-83268700-4283-11e9-80e3-1b60b23e4d5d.png)

Os valores acima incluem: os impostos (com isenção de PIS, COFINS e ISS), INSS e IRPF do pró-labore de 28% do faturamento (simples) e de um salário mínimo R$ 998 (Lucro presumido) e também o valor do contador R$89,90 para simples e R$179,90 para LP

**Comment by copercini** [comment]
Quem quiser ver os cálculos detalhados, atualizei as tabelas com as regras de 2018, arrumei o cálculo de imposto de renda para pessoa física e fiz 2 versões: uma para empresas nacionais e outra para exportação de serviços (com isenção de PIS, COFINS e ISS)

[Simples_Nacional_2018_vs_Lucro_Presumido.xlsx](https://github.com/DyegoCosta/trabalhando-remoto/files/2949067/Simples_Nacional_2018_vs_Lucro_Presumido.xlsx)
[Simples_Nacional_2018_vs_Lucro_Presumido_Exportacao_Servicos.xlsx](https://github.com/DyegoCosta/trabalhando-remoto/files/2949068/Simples_Nacional_2018_vs_Lucro_Presumido_Exportacao_Servicos.xlsx)


OBS1. apenas o anexo III (V que vira III, entra como III) que é o que interessa para o repo foi atualizado, os outros anexos estão com os cálculos de 2017
OBS2. não sou contador então pode ocorrer dos cálculos não estarem 100% corretos, ficarei feliz de corrigir qualquer falha que alguém encontrar
OBS3. essa tabela é da Contabilizei com os cálculos de 2017 e pública, como fiz modificações, retirei o logotipo pois caso algo esteja errado não reflete sobre eles. Mas tudo bem colocar novamente, só não quero problemas

Tomara que o governo não mude as regras tão logo, assim a tabela funciona por mais tempo hehe

**Comment by fforbeck** [comment]
Fala @copercini. Valeu por compatilhar a planilha. Eu dei uma olhada rapida e aparentemente nao tem o valor da GPS (11%)  do pro-labore. Alem disso no LP tem a gps + o fpas 20% do pro-labore. Se vc ja incluiu isso desconsidere essa msg. Valeus!

**Comment by copercini** [comment]
Fala @fforbeck, isso está nas colunas INSS (Pró-labore), 11% para simples e 31% (11% + 20%) no LP

Mas seu comentário foi muito importante, pq percebi que o teto do cálculo estava desatualizado, atualmente é R$5.882,92 (então o valor máximo da GPS (11%) será R$642,34), e estava R$5.531,31

Mais uma vantagem do simples, quando se aposentar vai ser com o valor que contribuiu, perto do teto, enquanto lucro presumido, será apenas um salário mínimo

Já atualizei as planilhas com o novo teto no post acima =)

**Comment by brunobarretosa** [comment]
@copercini valeu cara!! Tu é um gênio! hahaha obrigado!

**Comment by rpagliuca** [comment]
Pessoal, codei uma simples calculadora para simular o total de imposto a ser pago por desenvolvedores PJs que emitem nota fiscal apenas **contra empresas de fora do Brasil**.

Utilizei a estratégia fiscal de forçar o Anexo 3 emitindo 28% do faturamento (fator R) na forma de pró-labore. Quem quiser ver o projeto, está aqui:
https://github.com/rpagliuca/calculadora-imposto-dev-remoto

Deixei a calculadora rodando no AWS Lambda aqui nesse endpoint:
https://ooh86uy8d7.execute-api.us-east-1.amazonaws.com/dev?faturamento-anual=120000
(API only! rsrs)

Quem quiser simular, é só acessar essa URL acima, alterando o R$120.000 (parâmetro `faturamento-anual`) ali no final da URL, trocando por qualquer outro valor de faturamento anual.

Agradeço muito qualquer feedback e possíveis correções no cálculo! Podem mandar PR lá no repo.

A regra específica do cálculo, e constantes que utilizei estão aqui nesse arquivo:
https://github.com/rpagliuca/calculadora-imposto-dev-remoto/blob/master/function1/fiscal.go

**Comment by rpagliuca** [comment]
Ahhh, faltou falar que a calculadora só calcula para Simples Nacional. E só para empresas que somente prestam serviço para clientes de fora do Brasil, já que eu abato o PIS, COFINS e ISS.

**Comment by renatonascalves** [comment]
Legal!

**Comment by gabrielponto** [comment]
@rpagliuca Parabéns pelo projeto, é muito bom para tomar a decisão de se tornar PJ com mais clareza.

**Comment by rpagliuca** [comment]
Obrigado, pessoal!

Acabei de fazer umas validações do meu projeto contra a planilha `Simples_Nacional_2018_vs_Lucro_Presumido_Exportacao_Servicos.xlsx` criada pelo @copercini no ano passado, e os valores bateram certinho!

Só precisei atualizar o teto do INSS na planilha dele antes de poder comparar, pois hoje o teto já está mais alto.

Testei para faturamentos anuais de R$120.000, R$240.000 e R$500.000, e o resultado foi correto em todos os casos.

**Comment by renatonascalves** [comment]
@rpagliuca Poderia compartilhar a sua versão atualizada aqui?

Acho que a pocentagem do anexo III mudou também, não?

**Comment by rpagliuca** [comment]
@renatonascalves Segue arquivo atualizado, mas com a ressalva de que editei no Libreoffice Calc e o formato é XLSX, então há a possibilidade de perda de formatação (ou pior, rs): [Simples_Nacional_2018_vs_Lucro_Presumido_Exportacao_Servicos - Atualizado INSS 2020.xlsx](https://github.com/DyegoCosta/trabalhando-remoto/files/4870507/Simples_Nacional_2018_vs_Lucro_Presumido_Exportacao_Servicos.-.Atualizado.INSS.2020.xlsx)


Se preferir, faça você mesmo a atualização a partir da planilha original:

O novo teto da base do INSS em 2020 é R$6.101,06. Multiplicando por 11% temos R$671.12 de desconto máximo por mês.

Então a única coisa que fiz foi re-exibir a aba `INSS` (pois por padrão está invisível), e em seguida alterei a célula `INSS.C17` de `642.34` para `671.12`.

Quanto aos percentuais do Anexo III, pelo que vi já estão na última versão na planilha. Comparei a planilha com essa fonte: https://blog.contabilizei.com.br/contabilidade-online/anexo-3-simples-nacional/

Ahh, e se você quiser validar a planilha contra o projeto https://github.com/rpagliuca/calculadora-imposto-dev-remoto, você precisa primeiro zerar também a coluna `Cálculo.J9:J20`, para desconsiderar custos com contador.

**Comment by renatonascalves** [comment]
@rpagliuca Obrigado! Estou vendo ainda mas acho que tem algo errado na forma como está calculando ou algo errado na planilha.

Usando o anexo III com os valores mais recentes, com deduções:
![Screen Shot 2020-07-03 at 16 05 51](https://user-images.githubusercontent.com/19148962/86493247-2936b400-bd47-11ea-95f4-83b11dc0649e.png)

Eu tenho valores de aliquotas diferentes da sua. Coloquei como exemplo: R$ 320.000 anual.

![Screen Shot 2020-07-03 at 16 06 10](https://user-images.githubusercontent.com/19148962/86493262-4075a180-bd47-11ea-8b26-d2e7469084e5.png)

Na sua ferramenta, o imposto total anual é de: 10.40%. Os impostos do IR e INSS estão corretos, mas o da DAS não. O seu calculo está dando menos que o cálculo da tabela.

https://ooh86uy8d7.execute-api.us-east-1.amazonaws.com/dev?faturamento-anual=312000

**Comment by rpagliuca** [comment]
@renatonascalves Pelo valor aí de 8,20% de alíquota, acho que você está usando a planilha `Simples_Nacional_2018_vs_Lucro_Presumido.xlsx` em vez da
`Simples_Nacional_2018_vs_Lucro_Presumido_Exportacao_Servicos.xlsx` (tem as 2 aí em cima nos comentários, dá uma olhada).

No caso da Exportação de Serviços (emissão de nota fiscal apenas para clientes de outros países) a alíquota tem um desconto de 50.9% (desconta PIS, COFINS e ISS), então a alíquota nessa faixa de 312000 anual cai quase a metade.

Essa calculadora (https://ooh86uy8d7.execute-api.us-east-1.amazonaws.com/dev?faturamento-anual=312000) só funciona pra esse caso de exportação de serviços, então ela já aplica essa isenção de PIS, COFINS e ISS automaticamente.

**Comment by renatonascalves** [comment]
Entendi! Estou usando a segunda: Simples_Nacional_2018_vs_Lucro_Presumido_Exportacao_Servicos.xlsx

Por isso achei estranho. Acho que deve ter algo errado na planilha então. Vou tentar investigar.

**Comment by rpagliuca** [comment]
Isso! Baixa a planilha de novo daqui do comentário lá do ano passado: https://github.com/DyegoCosta/trabalhando-remoto/issues/54#issuecomment-471227716

Pq eu acabei de baixar as 2 versões, e consegui reproduzir essa diferença. Rodando o arquivo 1 a alíquota fica em 8.2 alí na planilha, e rodando o arquivo 2 cai pra perto de 4%

**Comment by renatonascalves** [comment]
Acho que escontrei o problema. Os dados anteriores do anexo já foram colocado com esse desconto.

![Screen Shot 2020-07-03 at 16 26 43](https://user-images.githubusercontent.com/19148962/86493984-fe019400-bd49-11ea-9ce0-12bd95bbb4eb.png)

**Comment by renatonascalves** [comment]
Tudo certo! Obrigado e desculpa a confusão!

Quem tiver lendo isso aqui no futuro, recomendo baixar essa versão: https://github.com/DyegoCosta/trabalhando-remoto/issues/54#issuecomment-653608924 Pois está atualizada com os novos valores do INSS.

E essa tabela bate 100% com https://ooh86uy8d7.execute-api.us-east-1.amazonaws.com/dev?faturamento-anual=312000

o/

**Comment by heliomeiralins** [comment]
Pessoal, no cálculo do fator R dá para entrar o custo do INSS também.
Saquem o link do da [contabilizei](https://www.contabilizei.com.br/calculadora-fator-r/)

Eu havia lido em um blog post* que o CSLL também pode entrar nesse cálculo. Contudo ainda não consegui confirmar em nenhum lugar.

* sorry não achei o link desse post

EDIT: conversei com um contador e ele falou que o INSS não entra no cálculo mesmo. é como se fosse um custo da PF e não da PJ.

**Comment by raphaeljlps** [comment]
@heliomeiralins provavelmente isso já entra nos cálculos do pessoal, o valor ali é bruto não o liquido que você recebe, dps é descontado o IR/INSS (no caso de pró-labore). Sobre a CSLL, são 3,5% do valor da DAS, um valor insignificante em relação ao faturamento. 

Exemplo: 

120.000 / ano
R = 28%
-> 120.000 * 0,28 = 33600 -> 2800 mês bruto (cadastra isso como pró-labore e retira o líquido) -> R$ 2.447,90
DAS ~3,05% do Faturamento = (120000*0,0305)/12 -> ~300/mês que implica CSLL de R$ 10,50/mês

**Comment by KamilTy** [comment]
Informações bem úteis.
Talvez alguém pode me ajudar com com umas informações referente utilização do $$ da conta PJ para pagar menos impostos.
Tenho Me no simples, posso retirar até 32% de faturamento com lucro sem imposto e para ficar no anexo III preciso ter gasto com pró-labore acima de 28%. Como posso usar o resto para pagar menos impostos, pagar umas despesas relacionadas tudo bem, mas ainda sobra bastante :)
Meu contador não é muito comunicativo.

**Comment by raphaeljlps** [comment]
@KamilTy somente no Lucro Real você pode abater despesas e depreciação do Faturamento para diminuir o Lucro e pagar menos impostos. No Simples Nacional (e Lucro Presumido) o imposto é calculado sobre o valor Bruto. No simples você pode tirar bem mais que 32% como lucro, não existe esse limite. Entretanto você tem que retirar um valor que seja somente lucro, geralmente se tira a antecipação de lucros depois de pago todos os impostos, taxas e pró-labore.

**Comment by heliomeiralins** [comment]
> Informações bem úteis.
> Talvez alguém pode me ajudar com com umas informações referente utilização do $$ da conta PJ para pagar menos impostos.
> Tenho Me no simples, posso retirar até 32% de faturamento com lucro sem imposto e para ficar no anexo III preciso ter gasto com pró-labore acima de 28%. Como posso usar o resto para pagar menos impostos, pagar umas despesas relacionadas tudo bem, mas ainda sobra bastante :)
> Meu contador não é muito comunicativo.

@KamilTy : Só complementado a resposta do @raphaeljlps, a estratégia no simples/lucro presumido é deixar a empresa só como um proxy pra vc emitir nota. Qualquer tipo de despesa e patrimônio sairia da sua distribuição de lucros que já não é taxada, então a regra geral é pagar tudo pela PF mesmo para simplificar a contabilidade da empresas.

Se um dia começarem a taxar distribuição de lucros aí vc pode tentar encaixar tudo que é relativo a empresa como despesa, mas mesmo nesse cenário hipotético é bom ter bastante cuidado pra não misturar despesa PF com PJ. Isso pode dar treta.

**Comment by copercini** [comment]
Uma possível mudança na tributação vai taxar em 20% as transferências da conta PJ para conta PF acima de R$20000/mês

Então quem tem dinheiro na PJ, recomendo transferir antes da aprovação disso 
https://agenciabrasil.ebc.com.br/economia/noticia/2021-06/proposta-de-reforma-do-ir-preve-taxa-de-20-sobre-lucros-e-dividendos

**Comment by fforbeck** [comment]
> Uma possível mudança na tributação vai taxar em 20% as transferências da conta PJ para conta PF acima de R$20000/mês
> 
> Então quem tem dinheiro na PJ, recomendo transferir antes da aprovação disso 
> https://agenciabrasil.ebc.com.br/economia/noticia/2021-06/proposta-de-reforma-do-ir-preve-taxa-de-20-sobre-lucros-e-dividendos

Exatamente @copercini e tem uma boa chance de passar essa.  Se passar, o lance vai ser jogar todas as despesas possíveis na empresa.

**Comment by RangelReale** [comment]
@copercini pelo que eu entendi isso não vai valer para o simples (mas não sei sobre o lucro presumido), talvez seja mais uma vantagem do simples?

**Comment by copercini** [comment]
@RangelReale pois é, como é proposta, tudo pode mudar ainda, mas por enquanto o simples está de fora mas o lucro presumido incluso

**Comment by RangelReale** [comment]
Eu estou estudando esses cálculos, mas não estou achando de onde vem a informação de, no caso de usar o prolabore de 28%, a aliquota do INSS é 11%.
Segundo [esse link](https://www.contabilizei.com.br/contabilidade-online/inss-autonomo/), INSS de 11% é só se for pago sobre o valor do salário minimo (1100 hoje), não pode pagar 11% sobre um valor maior que esse. Senão seria a aliquota de 20%.

Tem alguma coisa que eu não entendi?

**Comment by RangelReale** [comment]
Será que seria [isso](https://www.contabilizei.com.br/contabilidade-online/como-funciona-o-recolhimento-do-inss-para-pessoa-juridica/)?
Quem é PJ não é um contribuinte individual/autonomo, é um tipo totalmente diferente, PJ?

**Comment by copercini** [comment]
@RangelReale individual/autônomo paga como pessoa física. Quem é PJ paga como pessoa jurídica (empresa), o valor depende do regime tributário escolhido, no geral a empresa paga 20% de INSS sobre o salário do empregado e o próprio empregado paga 11% de INSS sobre o salário. 

Mas no simples os 20% que a empresa pagaria, já estão inclusos no valor do Simples, por isso que só fazem o cálculo com 11% (parte que o empregado paga). No lucro presumido paga-se os 20%+11% 

espero ter ajudado e não confundido ainda mais kkkk

**Comment by RangelReale** [comment]
@copercini sim, buscando na internet é dificil achar essa informação de INSS para PJ, só fala de CLT e autônomo, obrigado pela explicação!
